app/Models/Campaign.php

```diff
@@ -580,4 +581,14 @@ public function follower(): int
     {
         return (int) $this->follower;
     }
+
+    /**
+     * @return mixed
+     */
+    public function users()
```

Comment:
This method name is very confusing to me, since you'd expect the campaign users to be its members. Also the other relations are all placed in the CampaignRelations model, to keep the campaign model clean.